src/modules/Dashboard/features/Applet/AddUser/LinkGenerator/LinkForm/LinkForm.test.tsx

```diff
@@ -0,0 +1,141 @@
+import { generatePath } from 'react-router-dom';
+import { fireEvent, screen, waitFor } from '@testing-library/react';
+import mockAxios from 'jest-mock-axios';
+
+import { page } from 'resources';
+import { renderWithProviders } from 'shared/utils';
+import { mockedAppletData } from 'shared/mock';
+
+import { LinkForm } from './LinkForm';
+
+const dataTestid = 'dashboard-add-users-generate-link';
+const link = 'https://example.com/join/8bac1a7e-9c72-472a-a722-21ece63af1e1';
+const setInviteLink = jest.fn();
+const getProps = (requireLogin = false) => ({
+  setInviteLink,
+  inviteLink: {
+    link,
+    requireLogin,
+  },
+});
+
+Object.assign(navigator, {
```

Comment:
Please add the logic to revert `navigator` Object to its initial value. Otherwise it could affect other tests where `navigator` used as well.